### PR TITLE
add arc60h

### PR DESCRIPTION
**Pull request by Vertex-kb**
## Description
add arc60 hotswap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/vertex/arc60h/rules.mk [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+SLEEP_LED_ENABLE = no
+
+
+DEFAULT_FOLDER = vertex/arc60h
+
+
+
```
Comment (zvecr) [inline keyboards/vertex/arc60h/rules.mk:25]:
```suggestion

```

File: keyboards/vertex/arc60h/chconf.h [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/* Copyright 2022 vertex
```
Comment (zvecr) [inline keyboards/vertex/arc60h/chconf.h:1]: This file should be removed

File: keyboards/vertex/arc60h/halconf.h [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+/*
+ * This file was auto-generated by:
+ *    `qmk chibios-confmigrate -i keyboards/bt66tech/bt66tech60/halconf.h -r platforms/chibios/common/configs/halconf.h`
+ */
+
+#pragma once
+
+#define HAL_USE_PWM TRUE
+
```
Comment (zvecr) [inline keyboards/vertex/arc60h/halconf.h:25]:
```suggestion

```

File: keyboards/vertex/arc60h/mcuconf.h [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ * This file was auto-generated by:
+ *    `qmk chibios-confmigrate -i keyboards/bt66tech/bt66tech60/mcuconf.h -r platforms/chibios/STM32_F103_STM32DUINO/configs/mcuconf.h`
+ */
+
+#pragma once
+
+#include_next <mcuconf.h>
+
+#undef STM32_PWM_USE_TIM1
+#define STM32_PWM_USE_TIM1 TRUE
```
Comment (zvecr) [inline keyboards/vertex/arc60h/mcuconf.h:27]:
```suggestion
#undef STM32_SPI_USE_SPI2
#define STM32_SPI_USE_SPI2 TRUE
```

File: keyboards/vertex/arc60h/rules.mk [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+SLEEP_LED_ENABLE = no
```
Comment (zvecr) [inline keyboards/vertex/arc60h/rules.mk:19]: Would also need `WS2812_DRIVER = spi` to use the configured `config.h` options

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/vertex/arc60h/rules.mk (hunk 5 above)
Comment (Vertex-kb) [reply keyboards/vertex/arc60h/rules.mk]: is that so？

File: keyboards/vertex/arc60h/chconf.h (hunk 2 above)
Comment (Vertex-kb) [reply keyboards/vertex/arc60h/chconf.h]: Deleted.

File: keyboards/vertex/arc60h/config.h [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
```
Comment (fauxpark) [inline keyboards/vertex/arc60h/config.h:22]:
```suggestion
```

File: keyboards/vertex/arc60h/rules.mk [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# MCU name
+MCU = STM32F103
+
+# Bootloader selection
+BOOTLOADER = stm32duino
+
```
Comment (fauxpark) [inline keyboards/vertex/arc60h/rules.mk:6]:
```suggestion
```

File: keyboards/vertex/arc60h/info.json [hunk 8]
```diff
@@ -0,0 +1,80 @@
+{
+    "keyboard_name": "ARC60H",
+    "manufacturer": "vertex",
+    "url": "",
+    "maintainer": "EasonQian1, Vertex-kb",
+    "usb": {
+        "vid": "0x7374",
+        "pid": "0x9770",
+        "device_version": "0.0.1"
+    },
```
Comment (fauxpark) [inline keyboards/vertex/arc60h/info.json:10]:
```suggestion
    },
    "processor": "STM32F103",
    "bootloader": "stm32duino",
```

File: keyboards/vertex/arc60h/rules.mk [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+SLEEP_LED_ENABLE = yes
+WS2812_DRIVER = spi
+
+DEFAULT_FOLDER = vertex/arc60h
+
+
```
Comment (fauxpark) [inline keyboards/vertex/arc60h/rules.mk:24]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Please fix the line endings on all of these files. They should be Unix style (LF), not Windows (CRLF).

File: keyboards/vertex/arc60h/config.h [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+ *
+ * NKRO may prevent your keystrokes from being detected in the BIOS, but it is
+ * fully operational during normal computer usage.
+ *
+ * For a less heavy-handed approach, enable NKRO via magic key (LShift+RShift+N)
+ * or via bootmagic (hold SPACE+N while plugging in the keyboard). Once set by
+ * bootmagic, NKRO mode will always be enabled until it is toggled again during a
+ * power-up.
+ *
+ */
```
Comment (fauxpark) [inline keyboards/vertex/arc60h/config.h:65]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Line endings still need to be fixed.

**Comment by Vertex-kb** [comment]
> Line endings still need to be fixed.

Brother, I have fixed it.

**Review by fauxpark — APPROVED** [review]

**Comment by Vertex-kb** [comment]
@drashna @fauxpark @zvecr  May I ask whether the PR can be merged

**Pull request merged**